doc/spinel-protocol-src/spinel-prop-mac.md

```diff
@@ -124,3 +124,29 @@ Structure Parameters:
 * Type: Read-Write
 * Packed-Encoding: `b`
 
+### PROP 4867: SPINEL_PROP_MAC_SRC_MATCH_ENABLED  {#prop-mac-src-match-enabled}
+* Type: Write
+* Packed-Encoding: `b`
+
+Set to true to enable radio source matching or false to disable it.
```

Comment:
Let's document that these properties are available only under raw link API and tied to `SPINEL_CAP_MAC_RAW` capability.

Also another thing to update: `PHY_ENABLED` is always readable but writable only under raw link API (`SPINEL_CAP_MAC_RAW` capability).